tests/testsuite/package.rs

```diff
@@ -3410,6 +3411,29 @@ derp/not_target/foo.txt
 derp/target/foo.txt
 src/main.rs
 target/foo.txt
+",
+        )
+        .run();
+
+    // Untracked files shouldn't be included, if they are also ignored.
+    _ = fs::write(repo.workdir().unwrap().join(".gitignore"), "target/").unwrap();
+    git::add(&repo);
+    git::commit(&repo);
+    _ = fs::write(p.build_dir().join("untracked.txt"), "").unwrap();
```

Comment:
This test was added to further stress the `gitoxide` implementation, which I could otherwise have implemented wrongly. But even if that wasn't the case, it's good to see that only tracked files in `target` will be picked up.